api-client/src/endpoints/createShelterEndpoint.ts

```diff
@@ -6,14 +6,38 @@ import { JsonShelterType } from '../types'
 const SHELTER_URL = 'wohnraum.tuerantuer.org/wp-json/accommodations'
 
 export const SHELTER_ENDPOINT_NAME = 'shelter'
-type Params = { type: 'detail'; id: string; cityCode: string } | { type: 'list'; page: number; cityCode: string }
+type Params =
+  | { type: 'detail'; id: string; cityCode: string }
+  | { type: 'list'; page: number; cityCode: string; filter?: FilterProps }
```

Comment:
Does this have to be optional? I think logic would be a little bit easier if it is not and it should always be possible to set filters doing nothing as default. I think it is also never null/undefined.

Reply:
thats true, check last commit